### ISSUE TITLE
Always have a single space surrounding argument specification?

### ISSUE THREAD

**Issue by IndrajeetPatil** [r-lib/styler#881]
# observed 

Here I am deliberately adding a single, double, or triple spaces surrounding the argument assignment (`=`), but, when styled, `{styler}` preserves these spaces.

``` r
styler::style_text(
  'df <- dplyr::rename(df,
  "xValues" =   "Time",
  "xUnit" = "TimeUnit",
  "yValues"   = "simulationValues",
  "yUnit" = "unit",
  "yDimension"  = "dimension"
)'
)
#> df <- dplyr::rename(df,
#>   "xValues" =   "Time",
#>   "xUnit" = "TimeUnit",
#>   "yValues"   = "simulationValues",
#>   "yUnit" = "unit",
#>   "yDimension"  = "dimension"
#> )
```

<sup>Created on 2021-12-23 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

# expected

I would have expected that it would instead always have a single space surrounding assignment. Can this be supported?

```r
df <- dplyr::rename(df,
  "xValues" = "Time",
  "xUnit" = "TimeUnit",
  "yValues" = "simulationValues",
  "yUnit" = "unit",
  "yDimension" = "dimension"
)
```

# No changes in case alignment is detected

Of course, the feature request is relevant only in case no alignment is detected, since then the spaces are deliberate and shouldn't be changed.

``` r
styler::style_text(
  'df <- dplyr::rename(df,
  "xValues"    = "Time",
  "xUnit"      = "TimeUnit",
  "yValues"    = "simulationValues",
  "yUnit"      = "unit",
  "yDimension" = "dimension"
)'
)
#> df <- dplyr::rename(df,
#>   "xValues"    = "Time",
#>   "xUnit"      = "TimeUnit",
#>   "yValues"    = "simulationValues",
#>   "yUnit"      = "unit",
#>   "yDimension" = "dimension"
#> )
```

<sup>Created on 2021-12-23 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

**Comment by lorenzwalthert** [comment]
Thanks @IndrajeetPatil, 100% right. The problem was that quoted keys were not recognized. The following worked already:

```r
styler::style_text(
  'df <- dplyr::rename(df,
  xValues    = "Time",
  xUnit      = "TimeUnit",
  yValues    = "simulationValues",
  yUnit      = "unit",
  yDimension = "dimension"
)'
)
```
